### ISSUE TITLE
When trying to Change the build variant to "normalOptimized" then "c++ configuration error" found

### ISSUE THREAD

**Issue by Hidte09** [hrydgard/ppsspp#16989]
### Platform

Android

### Compiler and build tool versions

Clang+cmake in Android studio 

### Operating system version

Windows 10

### Build commands used

 ./b.sh --normaloptimized

### What happens

When trying to Change the build variant to "normalOptimized" then "c++ configuration error" found error occurred , like this
C/C++ Configuration Problem

FRESH PSP IMPORT.android.main | vrDebug|

arm64-v8a

Compiler exited with error code 1:

C:\Users\hidte\AppData\Local\Android\Sdk\ndk\

21.4.7075529\toolchains\/lvm\prebuilt\windows-x

86_64\bin\clang++.exe -xc++ --target-aarch64-none-linux-android21

--gcc-toolchain= C:/Users/hidte/AppData/Local/A

ndroid/Sdk/ndk/21.4.7075529/toolchains/llvm/pre

built/windows-x86_64

### PPSSPP version affected

V1.14.1

### Last working version

V1.14.1

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] Make sure to run `git submodule update --init --recursive` before building.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try deleting the build directory and running the build again.
- [X] Check GitHub Actions, which builds every merge and PR.
- [ ] Include logs and help us reproduce.

**Comment by ANR2ME** [comment]
As i remembered, even switching Build variant (ie. Debug <--> normalOptimized or ARMv7 <--> ARM64v8) from Android Studio can sometimes show errors during gradle sync (seems to be related to x86/x64)

**Comment by Hidte09** [comment]
What would be solution to fix the problem

**Comment by ANR2ME** [comment]
I'm not sure what causing the error nor how to fix it properly, i usually reopen the project after getting that error, and build it without changing the build variant (since the build variant is already selecting the one i was trying to select before when reopened)

**Comment by Hidte09** [comment]
![IMG_20230223_162823](https://user-images.githubusercontent.com/124633547/220985432-1e1fa73e-ae7e-4b66-bf7d-e3d5b34a4e27.png)
This is an actual error screenshot

**Comment by hrydgard** [comment]
Hm, Active ABI should probably be set to ARM64, but shouldn't really matter if you build an APK since it builds all. Doesn't look like you have a device connected.

Anyway, can you show what's on the build tab at the bottom?  You did do `git submodule update --init --recursive` right?

**Comment by Hidte09** [comment]
Yes I did git submodule update --init --recursive even though getting C/C++ Configuration Problem.
![IMG_20230223_162823](https://user-images.githubusercontent.com/124633547/221101164-89eb6aab-18da-4d41-947e-90f633040963.png)

**Comment by hrydgard** [comment]
You need to show some more of the actual errors for anybody to be able to diagnose the problem. This doesn't seem to happen for me.

**Comment by ANR2ME** [comment]
btw if you build it using `gradlew assembleNormalOptimized - - stacktrace` (like what we have in manual_generate_apk.yml at github action) does it gets error too?
